home/tests/test_api.py

```diff
@@ -281,6 +365,56 @@ def test_whatsapp_disabled(self, uclient):
         response = uclient.get(f"/api/v2/pages/{page.id}/?whatsapp=True")
         assert response.content == b""
 
+    def test_messenger_disabled(self, uclient):
+        """
+        It should not return the web body if enable_messenger=false
```

Comment:
Web body? (Also other tests below.)

Reply:
Sorry this was coppied from the whatsapp version of the test, I thought it was intentional